File: backend/database/models/lowprice.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const lowPrice = sequelize.define('lowPrice', {
    price: {
      type: DataTypes.DECIMAL,
      allowNull: false,
      validate: {
        notNull: {
          msg: 'Value can not be null.'
        },
        isDecimal: {
          args: true,
          msg: 'Price must be a decimal.',
        },
      }
    }
  }, {
    freezeTableName: true,
    tableName: 'lowPrice',
  });
  lowPrice.associate = function(models) {
    lowPrice.belongsTo(models.position, {foreignKey: 'positionId'});
  };
  return lowPrice;
};

File: backendSharp/Models/LowPrice.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;


namespace Trowoo.Models
{
    public class LowPrice
    {
        public int Id {get; set;}
        [Required]
        public decimal Price {get; set;}

        [Required]
        [ForeignKey("PositionId")]
        public int PositionId {get; set;}
    }
}